user-guide/Reference/DataMiner_Tools/Standalone_Elastic_Backup_Tool.md

```diff
@@ -72,7 +72,7 @@ To take a backup/snapshot, use the following arguments:
   - If none is defined and only one repository is found in the Elasticsearch cluster, then that one will be used.
   - If none is defined and none can be found, no backup will be taken.
 
-- `--snapshotname` or `-n`: The (lowercase) name of the snapshot to be taken.
+- `--SnapshotName` or `-n`: The (lowercase) name of the snapshot to be taken.
```

Comment:
I am a bit confused about the casing here, as it is specifically mentioned that this is the lowercase name of the snapshot.

Reply:
The value of the argument should indeed be lowercase. It's the name of the argument that had the wrong casing.